src/jwt_mock.js

```diff
@@ -0,0 +1,12 @@
+/**
+   Mock empty implementation of jsonwebtoken's .sign() method.
+   This is used by webpack in browser build in order to avoid
+   the node-only dependency breaking builds downstream.
+*/
+const jwt = {
+  sign: (/* value, secret */) => {
+    throw new Error('JWT support not implemented.');
+  },
```

Comment:
Should we implement a mock `verify` as well?

Reply:
Main sdk doesn't need it, it's only used in tests, so I think we're safe without it. It should anyway throw error if one attempts to call it and is missing.